api/schema_responses/views.py

```diff
@@ -94,13 +94,7 @@ class SchemaResponseDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIVie
     view_category = 'schema_responses'
     view_name = 'schema-responses-detail'
 
-    def get_serializer_class(self):
```

Comment:
referencing the parent here is an SQL query that isn't committed in the transaction where the `serilaizer_class` is  AttributeError not found is thrown. It's hard to describe but in the context this error is thrown it can't make any SQL queries, so I'm just removing this function for now.